modular_exponential/modular_exponential.py

```diff
@@ -0,0 +1,21 @@
+def mod_expo(base, power, mod):
```

Comment:
Please fix the function name.. modular_exponential or mod_exponential is better. 
No one uses expo for exponential therefore it can be misleading. 
I hope you understand.